src/SDKs/ApplicationInsights/Management.ApplicationInsights/Microsoft.Azure.Management.ApplicationInsights.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <Description>Microsoft Azure Application Insights Library</Description>
+    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <AssemblyName>Microsoft.Azure.Management.ApplicationInsights</AssemblyName>
+
+    <PackageId>Microsoft.Azure.Management.ApplicationInsights</PackageId>
+    <PackageTags>Management.ApplicationInsights;</PackageTags>
```

Comment:
Add < PackageReleaseNotes > in .csproj
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Language/Microsoft.CognitiveServices.Language/Microsoft.CognitiveServices.Language.csproj#L11

Reply:
Added, thanks!